src/corehost/cli/json/casablanca/include/cpprest/details/SafeInt3.hpp

```diff
@@ -5594,6 +5594,11 @@ template < typename T, typename E = SafeIntDefaultExceptionHandler > class SafeI
         *this = SafeInt< T, E >( (U)u );
     }
 
+    SafeInt(const SafeInt< T, E >& u) SAFEINT_NOTHROW
+    {
+        *this = u;
```

Comment:
This basically defers the implementation to `operator=`, which is defined later in this file.

Reply:
I assume the compiler forced you to add a copy constructor... in that case this look good.

Reply:
Thanks for the review!